### DISCUSSION TITLE
这几天nextchat用不了了，显示无效请求

### DISCUSSION THREAD

**Discussion by guohui1992** [BurnHair/docs#113] [技术支持]
![image](https://github.com/BurnHair/docs/assets/22585034/1cddef18-cadc-4825-99b6-d40edc5ad367)

**Comment by BennyThink** [comment]
需要配置一下

https://github.com/BurnHair/docs/blob/master/lobe-chat%20%E5%92%8C%20ChatGPT-Next-Web%20%E4%BD%BF%E7%94%A8.md

**Comment by guohui1992** [comment]
可以了，非常感谢
